### DISCUSSION TITLE
ChromeDriver v 107 is not getting installed in Node V 14.7.0

### DISCUSSION THREAD

**Discussion by rakeshkumarbarik** [giggio/node-chromedriver#398] [General]
`Extracting zip contents to C:\Users\pnl0sk91\AppData\Local\Temp\107.0.5304.62\chromedriver.
ChromeDriver installation failed TypeError: fs.rmSync is not a function
    at copyIntoPlace (C:\repos\gall-test\node_modules\chromedriver\install.js:314:6)
    at install (C:\repos\gall-test\node_modules\chromedriver\install.js:73:11)`

**Comment by giggio** [comment]
Version 14.7.0 is not LTS and is no longer supported. Migrate to an LTS version, which starts for 14 in 14.15.0: https://nodejs.org/en/blog/release/v14.15.0/
